instrumentation/cassandra/cassandra-4.4/library/src/main/java/io/opentelemetry/instrumentation/cassandra/v4_4/CassandraAttributesExtractor.java

```diff
@@ -74,4 +87,38 @@ public void onEnd(
     }
     attributes.put(SemanticAttributes.DB_CASSANDRA_IDEMPOTENCE, idempotent);
   }
+
+  private static void updateServerAddressAndPort(AttributesBuilder attributes, Node coordinator) {
+    EndPoint endPoint = coordinator.getEndPoint();
+    if (endPoint instanceof DefaultEndPoint) {
+      InetSocketAddress address = ((DefaultEndPoint) endPoint).resolve();
+      attributes.put(SemanticAttributes.SERVER_ADDRESS, address.getHostString());
+      attributes.put(SemanticAttributes.SERVER_PORT, address.getPort());
+    } else if (endPoint instanceof SniEndPoint && proxyAddressField != null) {
+      SniEndPoint sniEndPoint = (SniEndPoint) endPoint;
+      Object object = null;
+      try {
+        object = proxyAddressField.get(sniEndPoint);
+      } catch (Exception e) {
+        logger.fine(
+            "Error when accessing the private field proxyAddress of SniEndPoint using reflection.");
```

Comment:
(this is probably more common in the codebase)
```suggestion
        logger.fine(e.getMessage(), e);
```

Reply:
@trask actually doing it like this doesn't make sense because exception message is already logged along with the exception. Also `fine` method doesn't have an overload that would accept these arguments, you'd need to use `log.log(Level.FINE, e.getMessage(), e);`